Library/Homebrew/bottle_version.rb

```diff
@@ -63,3 +65,25 @@ def self._parse spec
     super
   end
 end
+
+class BottlePkgVersion < PkgVersion
```

Comment:
Basically, we should use `BottlePkgVersion` only. `BottleVersion` only serves as the internal API. We do the same thing in `Version` and `PkgVersion`.